activation/poetdb.go

```diff
@@ -127,16 +136,20 @@ func (db *PoetDb) publishPoetProofRef(key poetProofKey, poetProofRef []byte) {
 	delete(db.poetProofRefSubscriptions, key)
 }
```

Comment:
you can shorten this to Get the poet part is implied
also you should use this method inside GetMembershipByPoetProofRef
i would change "GetMembershipByPoetProofRef" to GetAsMessage or something that implies the conversion from bytes